rsconnect/environment.py

```diff
@@ -153,6 +157,14 @@ def create_python_environment(
             # that didn't support environment.python.requires
             environment.python = override_python_version
 
+        if package_manager is not None:
+            if package_manager not in ("pip", "uv"):
+                raise RSConnectException("Unsupported package manager: %s" % package_manager)
```

Comment:
Changing `package_manager` to a `Literal['uv', 'pip']` might remove the need for this check.